src/parse/asp/config.go

```diff
@@ -94,7 +94,7 @@ func (i *interpreter) newConfig(state *core.BuildState) *pyConfig {
 	return &pyConfig{base: base}
 }
 
-func resolveSelf(values []string, subrepo string) []string {
+func resolvePluginValue(values []string, subrepo string) []string {
```

Comment:
I think this needs updating? Isn't it changing `//foo:bar` to `/////foo:bar` for default values? I think we want to do something like:

```
if core.LooksLikeABuildLabel(v) && !strings.HasPrefix(v, "///") && !strings.HasPrefix(v, "@") {
       // convert to ///{subrepo}{v}
}
```

Reply:
Default values come from plugin config, which means the `subrepo` argument will be non-empty (i.e. `proto`), yielding `///proto//foo:bar` instead.